py_mind_memo/graphics.py

```
@@ -187,35 +199,51 @@ def get_text_size(self, node: Node, base_font, max_width: int = 250):
                 cached_photo, cached_data_hash = self.image_cache[node.id]
-                if cached_data_hash == current_data_hash:
-                    photo = cached_photo
+                if cached_data_hash == current_data_hash: photo = cached_photo
             
             if photo is None:
                 try:
                     photo = tk.PhotoImage(data=base64.b64decode(node.image_data))
                     self.image_cache[node.id] = (photo, current_data_hash)
-                except Exception:
-                    photo = None
+                except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)